dev-java/xsdlib/xsdlib-20130601.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="net.java.dev.msv:xsdlib:2013.6.1"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Sun XML Datatypes Library"
+HOMEPAGE="http://msv.java.net/"
+SRC_URI="https://repo1.maven.org/maven2/net/java/dev/msv/xsdlib/${PV/0601/.6.1}/xsdlib-${PV/0601/.6.1}-sources.jar"
```

Comment:
I'd prefer if we follow upstream's versioning scheme, so instead of 20130601 we have 2013.6.1. This should be trivially doable, since there are only a handful of consumers of this package. That all depend on on `dev-java/xsdlib:0`. So we merge this with upstream's versioning scheme, stabilize it, and then remove the old 20090415-r1 version.